ci.nix

```diff
@@ -42,9 +42,11 @@ dimension "Nixpkgs version" nixpkgsVersions (nixpkgsName: nixpkgs-pin:
       # TODO: can we merge this into the general case by picking an appropriate "cross system" to mean native?
       native = pkgs.recurseIntoAttrs {
         inherit (build) tests maintainer-scripts maintainer-script-cache;
-        hello = (pkgs.haskell-nix.hackage-package { name = "hello"; version = "1.0.0.2"; }).components.exes.hello;
-        iserv-proxy = pkgs.ghc-extra-packages.ghc865.iserv-proxy.components.exes.iserv-proxy;
-        ghc = pkgs.recurseIntoAttrs pkgs.haskell-nix.compiler;
+        sanity-checks = pkgs.recurseIntoAttrs {
```

Comment:
What are "sanity-checks"? Should we add a comment?

Reply:
We could take this change out.  I added it to help eval issues that were resolved by the change to `ci-lib.nix`.